torchrl/trainers/helpers/trainers.py

```diff
@@ -3,11 +3,13 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
-import argparse
-from argparse import ArgumentParser, Namespace
-from typing import Optional, Union
+from dataclasses import dataclass
+from typing import Optional, Union, List
 from warnings import warn
 
+from hydra import compose, initialize
```

Comment:
Can we decorate this with a try/except as I suggested in the previous PR?

Reply:
https://github.com/BoboBananas/rl/pull/3/commits/3e4a58a2220b0301563250bae00dcb9f5543df79